### DISCUSSION TITLE
help, i want to make a drum trainer program.

### DISCUSSION THREAD

**Discussion by wt208030** [community/community#112551] [Programming Help] [answered]
### Select Topic Area

Question

### Body

drum trainer program is education program. 

I'm working on a simple project. The function of the program I want to make is that when you add image score and then press the play button, one line of the score is displayed and a rectangular shape passes through the notes to show the progress of the song. And when you hit the drum, it checks whether the accuracy is the same as the score. I want to create a function that prints the results after a test.
I am using python and designing with pyqt. I would appreciate it if you could give me some reference material or advice.

like simplypiano app i wnat to make a program.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @wt208030 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by wt208030** [reply]
oops. sorry, thank you so much.
